lib/cocoapods-core/version.rb

```diff
@@ -46,14 +46,11 @@ class Version < Pod::Vendor::Gem::Version
     # @param  [String,Version] version
     #         A string representing a version, or another version.
     #
-    # @todo   The `from` part of the regular expression should be remove in
-    #         CocoaPods 1.0.0.
-    #
     def initialize(version)
       if version.is_a?(Version) && version.head?
         version = version.version
         @head = true
-      elsif version.is_a?(String) && version =~ /HEAD (based on|from) (.*)/
```

Comment:
A long long time ago

-Samuel E. Giddins

On Jun 6, 2015, at 6:14 PM, Kyle Fuller <notifications@github.com> wrote:

In lib/cocoapods-core/version.rb:

>      def initialize(version)
>        if version.is_a?(Version) && version.head?
>          version = version.version
>          @head = true
> -      elsif version.is_a?(String) && version =~ /HEAD (based on|from) (.*)/
Do we know what version from was used in?

—
Reply to this email directly or view it on GitHub.